CWE/events/italy.txt

```diff
@@ -455,6 +455,1043 @@ country_event = {
+		remove_country_modifier = backroom_boy
+		remove_country_modifier = great_reformer
+		remove_country_modifier = gauche_buffoon
+		remove_country_modifier = administrative_genius
+		remove_country_modifier = visionary_thinker
+		remove_country_modifier = competent_placeholder
+		remove_country_modifier = imperious_autocrat
+		remove_country_modifier = prince_of_terror
+		remove_country_modifier = efficient_sociopath
+		remove_country_modifier = pig_headed_isolationist
+		remove_country_modifier = ambitious_union_boss
+		remove_country_modifier = dour_economist
+		
+		add_country_modifier = landed_gentleman 
+		
+		any_pop = { 
+			ideology = { factor = 0.02 value = conservative } 
+		}
+		any_pop = { 
+			limit = { is_primary_culture = yes }
+			ideology = { factor = 0.02 value = nationalist } 
+		}
+		any_pop = { limit = { 
+		    OR = { region = PAP_752 region = SIC_753 region = SIC_754 region = SIC_763 region = SAR_765 } 
+			}
+			ideology = { factor = 0.04 value = traditionalist } 
+		}
+		
+		add_country_modifier = { name = political_strife duration = 365 }
+		
+		ai_chance = { factor = 0.1 }
+	}
+   
+}
+
+#Enrico Mattei and ENI
+country_event = {
+  	id = 8007751
+  	title = EVT_8007751_NAME
+  	desc = EVT_8007751_DESC
+  	picture = "enrico_mattei"
+	
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1953  
+		NOT = { year = 1990 }
+		
+		OR = { government = democracy government = hms_government }
+		is_vassal = no
+		war = no
+		
+		#Socialists & Progressives are not in power
+		NOT = { OR = { ruling_party_ideology = communist ruling_party_ideology = communist_social ruling_party_ideology = socialist ruling_party_ideology = progressive } }
+		
+		#Cold War ongoing
+		has_global_flag = blocsenabled
+	}
+
+	mean_time_to_happen = { months = 2  }
+
+	option = {
+		name = EVT_8007751_A
+		
+		petroleum = 1000
+		money = 500000
+		
+		random_country = {
+                       limit = { tag = USA government = democracy is_vassal = no }
+                       relation = { who = THIS value = -50 }
+        	}
+		random_country = {
+                       limit = { tag = ENG government = hms_government is_vassal = no }
+                       relation = { who = THIS value = -50 }
+        	}
+		random_country = {
+                       limit = { tag = NET government = hms_government is_vassal = no }
+                       relation = { who = THIS value = -25 }
+        	}
+		random_country = {
+                       limit = { tag = PER OR = { government = hms_government government = absolute_monarchy } is_vassal = no }
+                       relation = { who = THIS value = 50 }
+        	}
+		random_country = {
+                       limit = { tag = EGY government = presidential_dictatorship is_vassal = no }
+                       relation = { who = THIS value = 50 }
+        	}
+		random_country = {
+                       limit = { tag = JOR OR = { government = hms_government government = absolute_monarchy } }
```

Comment:
Please insert a check for is_vassal = no.

Reply:
I didn't put it because OTL Mattei ane ENI became close to the king if Jordan regardless of the protectorate, should I maybe use OR = { is_vassal = no is_vassal = yes)?

Reply:
I have a policy against allowing anyone other than the overlord to affect relations with vassals. It would make sense since foreign policy is handled by the overlord so there’s no way Italy can interact directly with a vassalised Jordan to boost relations between them.